pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -6854,7 +6878,30 @@ char_type:
     }
 }
 
+do_stmt:
+    DO expression_list
+    {
+        $$ = &tree.Do {
+            Exprs: $2,
+        }
+    }
 
+declare_stmt:
+    DECLARE variable_list
```

Comment:
use var_name_list:
var_name_list:
      var_name
      {...}
|.     var_name ',' var_name
      {....}

Reply:
lack of type
type:
    Any valid MySQL data type
you can use column_type